src/Compatibility/Microsoft.DotNet.ApiSymbolExtensions/Logging/ConsoleLog.cs

```diff
@@ -30,8 +32,20 @@ public virtual void LogWarning(string message) =>
             Console.WriteLine(message);
 
         /// <inheritdoc />
-        public virtual void LogWarning(string code, string message) =>
-            Console.WriteLine($"{code}: {message}");
+        public virtual void LogWarning(string code, string message)
+        {
+            string messageTextWithCode = $"{code}: {message}";
+
+            // Mimic MSBuild which logs suppressed warnings as low importance messages.
+            if (_noWarn.Contains(code))
+            {
+                LogMessage(MessageImportance.Low, messageTextWithCode)
```

Comment:
You missed a semicolon here:

```suggestion
                LogMessage(MessageImportance.Low, messageTextWithCode);
```